vt-digital-libraries-platform/iiif_s3_tools
# Removes legacy scripts and refactor/simplify generation of Batch job files for IIIF tiling.

**Removes legacy scripts and refactor/simplify generation of Batch job files for IIIF tiling.**
* * *

**Asana Ticket**: ([link](https://app.asana.com/0/1206697520698812/1206697305609589)) (:star:)

# What does this Pull Request do? (:star:)
Removes legacy scripts and refactor/simplify generation of Batch job files for IIIF tiling. Attempts to make environment variable names more descriptive and consistent

# What's the changes? (:star:)
* Cleans up the project directory by removing redundant versions of script.
* Refactors job file generation to simplify and clarify inputs

# How should this be tested?
* clone this repo and switch into the `wlh_20240117_refactor` branch
* replace `launch-task-generator-example.sh` with the `launch-task-generator.sh` file that I'll provide to reviewers via Slack
* cd into the `batch_task_json_generator/` directory
* (optional) launch a [Python virtual environment](https://docs.python.org/3/library/venv.html) to work in
* run `. ./launch-task-generator.sh`
* check output in the `json_files` directory

# Additional Notes:
* branch: `wlh_20240117_refactor`
* Developed and tested on macOS 14.3.1 
* I have no idea at all if this will work on Windows. There's a decent chance it will work in most Linux distros.

# Interested parties
@otokama 
@goynejennifer 

(:star:) Required fields
